applications/GeoMechanicsApplication/custom_elements/line_interface_element.h

```diff
@@ -0,0 +1,72 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Richard Faasse
+//
+
+#pragma once
+
+#include "includes/element.h"
+#include "includes/ublas_interface.h"
+#include "integration_scheme.h"
+#include "stress_state_policy.h"
+
+namespace Kratos
+{
+
+class KRATOS_API(GEO_MECHANICS_APPLICATION) LineInterfaceElement : public Element
+{
+public:
+    KRATOS_CLASS_INTRUSIVE_POINTER_DEFINITION(LineInterfaceElement);
+
+    using Element::GeometryType;
+    using Element::PropertiesType;
+    using BaseType = Element;
+
+    LineInterfaceElement() = default;
```

Comment:
The argument here is that (since it's default only, this doesn't hold when there is a real implementation), in this way we don't have to go to the source file just to see it's a default constructor, but I think it's a matter of preference.